mesonbuild/build.py

```diff
@@ -637,8 +637,9 @@ def get_install_dir(self) -> T.Tuple[T.List[T.Union[str, Literal[False]]], str,
                 outdirs[0] = default_install_dir
             else:
                 outdirs = [default_install_dir]
+            install_dir_names = [default_install_dir_name for i in outdirs]
```

Comment:
```suggestion
            install_dir_names = [default_install_dir_name for _ in outdirs]
```
or
```suggestion
            install_dir_names = [default_install_dir_name] * len(outdirs)

Reply:
Yep, should have fixed that at the same time as the other thing.

The second option is more efficient, which makes it an obvious shoo-in...